pkg/devfile/adapters/kubernetes/component/commandhandler.go

```diff
@@ -44,7 +44,7 @@ func (a *adapterHandler) Execute(devfileCmd devfilev1.Command) error {
 		switch status {
 		case remotecmd.Starting:
 			_ = log.SpinnerNoSpin(fmt.Sprintf("Executing the application (command: %s)", devfileCmd.Id))
-		case remotecmd.Stopped:
+		case remotecmd.Stopped, remotecmd.Errored:
 			if err != nil {
 				klog.V(2).Infof("error while running background command: %v", err)
 			}
```

Comment:
I propose this, so we can see when an application is terminated:

```
	var spin *log.Status
	startHandler := func(status remotecmd.RemoteProcessStatus, stdout []string, stderr []string, err error) {
		switch status {
		case remotecmd.Starting:
			spin = log.SpinnerNoSpin(fmt.Sprintf("Executing the application (command: %s)", devfileCmd.Id))
		case remotecmd.Stopped, remotecmd.Errored:
			if spin != nil {
				spin.End(status == remotecmd.Stopped)
				spin = nil
			}
			if err != nil {
				klog.V(2).Infof("error while running background command: %v", err)
			}
		}
	}
```

output:
```
↪ Deploying to the cluster in developer mode
 ✓  Waiting for Kubernetes resources [8s]
 ✓  Syncing files into the container [256ms]
 ✓  Building your application in container on cluster (command: build1) [2s]
 ✓  Building your application in container on cluster (command: build2) [983ms]
 •  Executing the application (command: run1)  ...
 ✓  Executing the application (command: run1) [2s]
 •  Executing the application (command: run2)  ...

Your application is now running on the cluster
 - Forwarding from 127.0.0.1:40001 -> 8080

```

Reply:
That's a good point, from the user perspective. I'll make the changes you suggested.